### DISCUSSION TITLE
How come this doesn't use the mongodb scala driver?

### DISCUSSION THREAD

**Discussion by esecules** [Kirill5k/mongo4cats#50] [Q&A] [answered]
Hi I was wondering why this repo doesn't use the [official mongodb scala driver](https://www.mongodb.com/docs/languages/scala/scala-driver/current/) and instead uses the java reactivestreams driver?

 Thanks,
Eric

**Comment by Kirill5k** [comment, marked as answer]
The main reason is because mongo-scala-driver is not compatible with Scala 3. Otherwise it would have been great to use it.
